packages/eslint-plugin-xstate/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint-plugin-xstate",
+  "version": "0.0.0",
+  "main": "dist/eslint-plugin-xstate.cjs.js",
+  "author": "with-heart <with.heart+git@pm.me>",
+  "license": "MIT",
+  "keywords": [
+    "eslint",
+    "eslintplugin"
+  ],
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "@typescript-eslint/utils": "^5.37.0"
+  },
+  "devDependencies": {
+    "@typescript-eslint/parser": "^5.37.0",
+    "eslint": "^8.0.0",
+    "typescript": "^4.3.5"
+  },
+  "peerDependencies": {
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0",
+    "typescript": "*"
+  },
+  "peerDependenciesMeta": {
+    "typescript": {
+      "optional": true
+    }
+  }
```

Comment:
I'm not entirely sure these dependencies are correct if we end up publishing this package. I somewhat copied the dependencies setup from [`@typescript-eslint/eslint-plugin`](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin), but they also don't have any of these as `devDependencies` which `yarn` whined about 🤷